inst/components/scss/sidebar.scss

```diff
@@ -2,7 +2,7 @@ $bslib-sidebar-bg: rgba(var(--bs-emphasis-color-rgb, 0,0,0), 0.05) !default;
 $bslib-sidebar-fg: var(--bs-emphasis-color, black) !default;
 $bslib-sidebar-toggle-bg: rgba(var(--bs-emphasis-color-rgb, 0,0,0), 0.1) !default;
 $bslib-sidebar-border: var(--bs-card-border-width, #{$card-border-width}) solid var(--bs-card-border-color, #{$card-border-color}) !default;
-$bslib-sidebar-column-sidebar: Min(calc(100% - var(--_icon-size)), var(--_sidebar-width));
+$bslib-sidebar-column-sidebar: Min(calc(100% - var(--_padding-icon)), var(--_sidebar-width));
```

Comment:
In desktop mode, if the sidebar is too wide for the container, this ensures that the sidebar doesn't expand _too_ far, leaving one collapse icon's worth of space in the main area. Which is useful if there are nested sidebars. (Slightly tangential to this PR but I didn't want to forget to fix this.)